syntax = "proto3";

package protocol;

message Heartbeat {
  uint32 node_id = 1;
  uint64 sent_at_ms = 2;
}
